manageiq-operator/pkg/helpers/miq-components/httpd_conf.go

```diff
@@ -65,6 +62,9 @@ Options SymLinksIfOwnerMatch
   RewriteRule .* ws://remote-console:3000%%{REQUEST_URI}  [P,QSA,L]
```

Comment:
Maybe this section should move up below the notifications section instead?

Reply:
I will try this. And test a bit more. 

This configuration is really hard to figure out what is really going on.

Reply:
See the updated PR. I moved the section below the notifications section. 

Tests I did:
- /openid-connect --> Not routed
- /ws/console --> redirected to remote console pods
- /api --> web services pods
- UI is still functional